src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -33,6 +33,8 @@ internal static class BenchmarkRunnerClean
 
         internal static readonly IResolver DefaultResolver = new CompositeResolver(EnvironmentResolver.Instance, InfrastructureResolver.Instance);
 
+        private static TaskbarProgress s_taskbarProgress;
```

Comment:
We should avoid having disposable static instances, please just pass the instance to the right method.

Reply:
Of course, much cleaner!